### PR TITLE
[GSIP 143] Turbojpegstatus

### PR DESCRIPTION
**Pull request by DBlasby**
This is a ModuleStatus implementation for LibTurboJpeg.

It's fairly simple, but gives decent status information on LibTurboJpeg.

### CODE REVIEW

File: src/extension/libjpeg-turbo/src/main/java/org/geoserver/map/turbojpeg/TurboJPEGMapResponse.java [hunk 1]
```diff
@@ -35,6 +35,10 @@
 
     private final static boolean DISABLE_TURBO = Boolean.getBoolean("disable.turbojpeg");
 
+    public static boolean isDisabled() {
+        return DISABLE_TURBO;
+    }
+
     /**
```
Comment (tbarsballe) [inline src/extension/libjpeg-turbo/src/main/java/org/geoserver/map/turbojpeg/TurboJPEGMapResponse.java]: Header still says 2014

File: src/extension/libjpeg-turbo/src/test/java/org/geoserver/map/turbojpeg/TurboJpegStatusTest.java [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    public void testTurboJpegStatus() {
+        
+        try{
+            Runtime.getRuntime().loadLibrary("turbojpeg");
+        }
+        catch (java.lang.UnsatisfiedLinkError e)
+        {
+            Runtime.getRuntime().loadLibrary("turbojpeg");
+        }
+        TurboJpegStatus status = new TurboJpegStatus();
```
Comment (tbarsballe) [inline src/extension/libjpeg-turbo/src/test/java/org/geoserver/map/turbojpeg/TurboJpegStatusTest.java]: Doesn't match coding conventions: Shoud be 
```
try {
    ...
} catch {
    ...
}
```
(spacing and bracket location)

File: src/extension/libjpeg-turbo/src/main/java/org/geoserver/map/turbojpeg/TurboJPEGMapResponse.java (hunk 1 above)
Comment (DBlasby) [inline src/extension/libjpeg-turbo/src/main/java/org/geoserver/map/turbojpeg/TurboJPEGMapResponse.java]: fixed

**Comment by tbarsballe** [comment]
Looks good after fixes. I see this is for GSIP-143, so I am hapy to merge after the Travis Build completes

**Pull request merged**